composer.json

```diff
@@ -16,7 +16,7 @@
     "php": ">=7.1",
     "ext-json": "*",
     "cultuurnet/hydra": "~0.1",
-    "cultuurnet/udb3": "~0.1",
+    "cultuurnet/udb3": "dev-feature/III-2961 as 0.1",
```

Comment:
Should this be reverted now that the other branch was merged?

Reply:
Yep, reverted it just now! 👍